### ISSUE TITLE
Does it support wildcard domains?

### ISSUE THREAD

**Issue by askz** [ncarlier/dynamic-ambassador#1]
Hi ! Thanks for sharing this,
I'm really interested in this kind of ambassador since it supports load balancing
I wonder if it support wilcard domains like docker run -e DOMAIN_NAME=*.app.toto.fr 
Thanks in advance.

**Comment by ncarlier** [comment]
no, it doesn't. But I think it's possible to do by replacing "hdr(Host)" by "hdr_end(Host)" here: https://github.com/ncarlier/dynamic-ambassador/blob/master/confd%2Ftemplates%2Fhaproxy.tmpl#L8. If you do so, you will be able to set DOMAIN_NAME to "app.toto.fr" to also handle subdomains. However, it will be the default behavior for all containers.

Regards
